### ISSUE TITLE
[BUG] KeyError: 'shortcuts'

### ISSUE THREAD

**Issue by krichbanana** [xenova/chat-downloader#97]
## Basic information
- Program version: 0.0.8, updated to 0.1.0 in the background while running.
- Python version: 3.9.5
- Operating system: Arch Linux, kernel 5.13rc5

## Describe the bug
A fatal KeyError exception occured while downloading from an airing livestream.

### If running from the command line, provide the following:
1. The command used (including the verbose tag, `-v`):
```
chat_downloader -o "${outname?}".json --message_groups "all" --indent 2 -- "${url?}"
```
2. Output from the above command (source doesn't match the running program):
```
Traceback (most recent call last):
  File "$HOME/.local/bin/chat_downloader", line 8, in <module>
    sys.exit(main())
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/cli.py", line 170, in main
    
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/chat_downloader.py", line 297, in run
    output, indent=indent, sort_keys=sort_keys, overwrite=overwrite)
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/sites/common.py", line 60, in __iter__
    class Remapper():
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 1141, in _get_chat_messages
    def _get_chat_messages(self, initial_info, params):
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 412, in _parse_item
    
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/sites/common.py", line 280, in remap
    _TESTS = [
  File "$HOME/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 377, in parse_runs
    index = url.find('=')
KeyError: 'shortcuts'
```

## Expected behavior
I expected the downloader to finish normally.

## Additional context/information
It may be related to the KeyError: isCustomEmoji issue as I had personally replaced (just) that field with an instance of .get(). Furthermore, it seems there might be another chance of failure, with `name = emoji['shortcuts'][0]` just a bit earlier (which matches the line number given when using the 0.0.8 source).

**Comment by xenova** [comment]
Sure thing let me check this out. Can you provide the URL you tested with?

**Comment by krichbanana** [comment]
It's still live for now: `https://www.youtube.com/watch?v=b2B4hC6Dh6w`

**Comment by xenova** [comment]
Okay thanks! I'll run it and try to replicate the bug. Either way, it would be an easy fix to just use a `try_get` on the emoji item, so I will add that.

Although, it would be nice to see what message caused that. From similar issues in the past, this anomaly occurs when the YouTube API returns an incomplete item.

Will keep you updated 👍

**Comment by xenova** [comment]
I fixed this in the latest commit, and I will push the changes to the latest pip version (i.e., make a new release) once I add a fix for the consent issue (#85 and #94).
